lib/vmdb/appliance.rb

```diff
@@ -127,7 +127,11 @@ def self.get_build
 
     def self.get_network
       retVal = {}
-      retVal[:hostname] = LinuxAdmin::Hosts.new.hostname
+
+      dns = LinuxAdmin::Dns.new
+      retVal[:hostname]      = LinuxAdmin::Hosts.new.hostname
+      retVal[:primary_dns]   = dns.nameservers[0]
+      retVal[:secondary_dns] = dns.nameservers[1]
```

Comment:
Does calling nameservers twice cause it to shell out twice?  Perhaps those should be put into a variable first.

```ruby
nameservers = LinuxAdmin::Dns.new.nameservers
retVal[:primary_dns]   = nameservers[0]
retVal[:secondary_dns] = nameservers[1]